spec/io/io_spec.rb

```diff
@@ -120,6 +130,24 @@
         expect(df[:age][5]).to eq(nil)
         expect(@expected).to eq(df)
       end
+
+      it "loads DataFrame from second Excel Spreadsheet" do
```

Comment:
I think you didn't add the case, you are trying to fix with your code changes.

Reply:
Yes, I understand what you said. And I now think I should add a new case cause the test file should have more than one title row in my PR and the default testcase usage is no need to change.
Is this the correct way to add it?
![image](https://user-images.githubusercontent.com/10494196/66268018-3af83300-e86b-11e9-96ec-dd1c84fd7b2a.png)

Reply:
Yes you can create separate `context` as well as add few cases (`it` block) in already present test case. 

Instead of writing `second excel spreadsheet` you can write the name of the excel file. Variable names must be able to define the value it contains.